sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/KeyAsyncClient.java

```diff
@@ -492,14 +496,108 @@ Duration getDefaultPollingInterval() {
         KeyImportRequestParameters parameters = new KeyImportRequestParameters()
             .setKey(importKeyOptions.getKey())
             .setHsm(importKeyOptions.isHardwareProtected())
-            .setKeyAttributes(new KeyRequestAttributes(importKeyOptions));
+            .setKeyAttributes(new KeyRequestAttributes(importKeyOptions))
+            .setKeyReleasePolicy(importKeyOptions.getKeyReleasePolicy());
         return service.importKey(vaultUrl, importKeyOptions.getName(), apiVersion, ACCEPT_LANGUAGE, parameters,
             CONTENT_TYPE_HEADER_VALUE, context.addData(AZ_TRACING_NAMESPACE_KEY, KEYVAULT_TRACING_NAMESPACE_VALUE))
             .doOnRequest(ignored -> logger.info("Importing key - {}", importKeyOptions.getName()))
             .doOnSuccess(response -> logger.info("Imported key - {}", response.getValue().getName()))
             .doOnError(error -> logger.warning("Failed to import key - {}", importKeyOptions.getName(), error));
     }
 
+    /**
+     * Exports the latest version of a key from the key vault. The export key operation may be used to import any key
+     * from the Azure Key Vault as long as it is marked as exportable and its release policy is satisfied.
+     *
+     * <p><strong>Code Samples</strong></p>
+     * <p>Exports a key from a key vault. Subscribes to the call asynchronously and prints out the newly exported key
+     * details when a response has been received.</p>
+     *
+     * {@codesnippet com.azure.security.keyvault.keys.keyasyncclient.exportKey#String-String}
+     *
+     * @param name The name of the key to be exported.
+     * @param environment The target environment assertion.
+     * @return A {@link Mono} containing the {@link KeyVaultKey exported key}.
+     * @throws NullPointerException If the specified {@code name} or {@code environment} are {@code null}.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<KeyVaultKey> exportKey(String name, String environment) {
+        try {
+            return exportKeyWithResponse(name, "", environment).flatMap(FluxUtil::toMono);
```

Comment:
Version is not actually required, but you send it as an empty string? Wouldn't null make more sense, or is that not idiomatic with Java? If null is okay, I wouldn't validate that it's non-null in the overload. If they use that one and still pass null, it should be as acceptable as calling this overload.

Reply:
Since it's a part of the path I thought it would be required. Is calling `keys/{key-name}/export` equivalent to calling `keys/{key-name}/{key-version}/export` with the latest key version?

Reply:
Yes. I asked about this even with the older APIs. Their resolver on the backend checks the operation against the last path part. All the other APIs with optional versions also list them as required in swagger. You could elide the double slash (e.g. "/keys/{key-name}//export") in the URL in that case, but in my experience with the KV APIs I haven't seen it make a difference.